Add conan_imports function

maksim-0 opened the pull request:
There is NO_IMPORTS argument in conan_cmake_install and we use it in our projects. Then in order to copy artifacts to specific import folder we need to call `conan imports`.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/conan-io/cmake-conan?pullRequest=353) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/conan-io/cmake-conan?pullRequest=353) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/conan-io/cmake-conan?pullRequest=353) it.</sub>

czoido commented:
Hi @maksim-0,
Thanks a lot for the contribution. Please, could you provide a little bit more background on why you do a `conan install` with -`--no-imports` for later calling `conan imports `? Shouldn't the result be the same than doing the imports in the `conan install` ?

maksim-0 commented:
@czoido 

I was using it like this:
```cmake
# PRODUCT_BUILD_DIR is outside CMAKE_BINARY_DIR
set(
    CMAKE_RUNTIME_OUTPUT_DIRECTORY
    ${PRODUCT_BUILD_DIR}/bin
)
set(
    CMAKE_LIBRARY_OUTPUT_DIRECTORY
    ${PRODUCT_BUILD_DIR}/lib
)
set(
    CMAKE_ARCHIVE_OUTPUT_DIRECTORY
    ${PRODUCT_BUILD_DIR}/lib
)

conan_cmake_configure(
    REQUIRES
        boost/1.72.0
    IMPORTS
        "bin, *.dll -> ./bin"
        "lib, *.so* -> ./lib"
        "licenses, LICENSE_1_0.txt -> ./licenses @ root_package=boost, folder=True"
)

conan_cmake_autodetect(CONAN_SETTINGS)

conan_cmake_install(
    PATH_OR_REFERENCE .
    BUILD never
    SETTINGS ${CONAN_SETTINGS}
)

conan_imports(
    PATH .
    IMPORT_FOLDER ${PRODUCT_BUILD_DIR}
)
```
But you raised a good question. I was absolutely sure that I can't specify absolute paths in IMPORTS "section" of  ``conan_cmake_configure``. I tried and it appears that I actually can. So I removed ``conan_imports`` and rewrote ``conan_cmake_configure``:
```cmake
conan_cmake_configure(
    REQUIRES
        boost/1.72.0
    IMPORTS
        "bin, *.dll -> ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}"
        "lib, *.so* -> ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}"
        "licenses, LICENSE_1_0.txt -> ${PRODUCT_BUILD_DIR}/licenses @ root_package=boost, folder=True"
)
```
I'm sorry for taking up your time. We don't really need ``conan_imports``.
